cursorless-talon/src/actions/actions.py

```diff
@@ -59,27 +59,27 @@ def cursorless_command(action_id: str, target: dict):
         else:
             return actions.user.cursorless_single_target_command(action_id, target)
 
-    def cursorless_vscode_command(command_id: str, target: dict):
-        """Perform vscode command on cursorless target"""
-        return vscode_command(command_id, target)
+    def cursorless_server_command(command_id: str, target: dict):
```

Comment:
This one is part of the [Cursorless public api](https://www.cursorless.org/docs/user/customization/#cursorless-public-api), so we can't really change it.  We could use a wrapper tho like we did for the knausj PR

Reply:
I would advise

1. Rename old `cursorless_vscode_command` action to `cursorless_ide_command`
2. Add new action `cursorless_vscode_command` that wraps `cursorless_ide_command`